### PR TITLE
added connector metrics

### PR DESCRIPTION
**Pull request by DFSko**

### CODE REVIEW

File: Api/Infrastructure/Metrics/Counters.cs [hunk 1]
```diff
@@ -29,15 +29,34 @@ public static class Counters
...
+            {
+                LabelNames = new [] {"step", "supplier", "errorDetails", "errorCode"}
+            });
         
         
-        public static readonly Histogram SupplierSearchResponseTimeDuration = Prometheus.Metrics.CreateHistogram(
-            ApplicationPrefix + "supplier_search_response_time_duration",
-            "Supplier search response time duration",
+        public static readonly Histogram SupplierRequestHistogram = Prometheus.Metrics.CreateHistogram(
+            ApplicationPrefix + "supplier_request_histogram",
```
Comment (kirillta) [inline Api/Infrastructure/Metrics/Counters.cs:44]: Probably we need one more low dash for consistency like `ApplicationPrefix + "_supplier_request_histogram"`

File: Api/Infrastructure/Metrics/Counters.cs [hunk 2]
```diff
@@ -29,15 +29,34 @@ public static class Counters
...
+        public const string RoomSelection = "room-selection";
+        public const string Evaluation = "evaluation";
+        public const string Booking = "booking";
+        public const string Cancellation = "cancellation";
+        public const string GettingDeadline = "getting-deadline";
+        public const string GettingInformation = "getting-information";
+        public const string ProcessAsyncResponse = "process-async-response";
         
         
         private const string ApplicationPrefix = "edo_";
```
Comment (kirillta) [inline Api/Infrastructure/Metrics/Counters.cs:62]: And remove the dash here

File: Api/Infrastructure/Metrics/Counters.cs [hunk 3]
```diff
@@ -29,15 +29,34 @@ public static class Counters
...
+
+
+        public const string WideAvailabilitySearch = "wide-availability-search";
+        public const string RoomSelection = "room-selection";
+        public const string Evaluation = "evaluation";
+        public const string Booking = "booking";
+        public const string Cancellation = "cancellation";
+        public const string GettingDeadline = "getting-deadline";
+        public const string GettingInformation = "getting-information";
+        public const string ProcessAsyncResponse = "process-async-response";
```
Comment (kirillta) [inline Api/Infrastructure/Metrics/Counters.cs:59]: Last three constants sound like verbs. What `getting-information` means at all?

**Review by kirillta — CHANGES_REQUESTED** [review]

File: Api/Infrastructure/Metrics/Counters.cs (hunk 1 above)
Comment (DFSko) [reply Api/Infrastructure/Metrics/Counters.cs]: Fixed

File: Api/Infrastructure/Metrics/Counters.cs (hunk 2 above)
Comment (DFSko) [reply Api/Infrastructure/Metrics/Counters.cs]: Fixed

File: Api/Infrastructure/Metrics/Counters.cs (hunk 3 above)
Comment (DFSko) [reply Api/Infrastructure/Metrics/Counters.cs]: Fixed

**Review by kirillta — APPROVED** [review]

**Pull request merged**